### ISSUE TITLE
Angular example app tests failing

### ISSUE THREAD

**Issue by calebfroese** [NextFaze/npm-modules#1]
The tests for the `angular-example-app` project are failing due undeclared variables in the following files:

- `packages/angular-example-app/src/app/app.module.ts`
- `packages/angular-example-app/src/app/child-component/child-component.component.ts`
- `packages/angular-example-app/src/app/query/query.component.ts`

Presumably there is a LoopBack dependency, some may say a  _Loopback seed_ project that the tests here are relying on? 🙃

**Comment by zbarbuto** [comment]
Yes - we removed the Loopback SDK that was being used when we transitioned to open source. I think we will need to add a simple SDK and just commit it or mock it in order to get the tests passing again.
